MapboxCoreNavigation/DistanceFormatter.swift

```diff
@@ -4,6 +4,7 @@ let metersPerMile: CLLocationDistance = 1_609.344
 let secondsPerHour = 60.0 * 60.0
 let yardsPerMile = 1_760.0
 let feetPerMile = yardsPerMile * 3.0
+let yardsPerMeter = 1.0936133
```

Comment:
It’s possible to consolidate this constant with `feetPerMeter` or `yardsPerMile`, relying on the fact that there are three feet in a meter.